.github/copilot-instructions.md

```diff
@@ -0,0 +1,137 @@
+# GitHub Pages Deploy Action
+
+Always follow these instructions first and only fallback to additional search and context gathering if the information in these instructions is incomplete or found to be in error.
+
+## Working Effectively
+
+### Bootstrap and build the repository:
+
+- Use Node.js version from `.node-version` file: `v22.11.0`
```

Comment:
Node versions shouldn't be specified here, it should just reference to use the node-version file.